### ISSUE TITLE
`nest start` always start in debug mode

### ISSUE THREAD

**Issue by katsanva** [nestjs/nest-cli#642]
## I'm submitting a...

<pre><code>
[ ] Regression 
[x] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior

`nest start` always start in debug mode due to unsafe checks
Options contain even not initialized values: 
```
[
  { name: 'config', value: undefined },
  { name: 'webpack', value: undefined },
  { name: 'debug', value: undefined },
  { name: 'watch', value: false },
  { name: 'watchAssets', value: false },
  { name: 'path', value: undefined },
  { name: 'webpackPath', value: undefined },
  { name: 'exec', value: undefined },
  { name: 'preserveWatchOutput', value: false }
]
```
So next check is false-positive
https://github.com/nestjs/nest-cli/blob/c7453668f29426b8faf57a71d0805d119238cf1e/actions/start.action.ts#L140
also, this one is suspicious 
https://github.com/nestjs/nest-cli/blob/c7453668f29426b8faf57a71d0805d119238cf1e/actions/start.action.ts#L62

## Expected behavior

`--debug` or `--debug-brk` options control that

## Minimal reproduction of the problem with instructions

Seems to be broken in https://github.com/nestjs/nest-cli/issues/592

Simply make `npm i` in https://github.com/nestjs/nest/tree/master/sample/12-graphql-schema-first or any other sample
```
➜  12-graphql-schema-first git:(master) ✗ ./node_modules/.bin/nest start
Debugger listening on ws://127.0.0.1:9229/c822fd22-351f-425b-b25a-cf354383e901
For help, see: https://nodejs.org/en/docs/inspector
```
## Environment

```

```

**Comment by kamilmysliwiec** [comment]
Fixed in 7.1.1 Sorry about that!

**Comment by atjeff** [comment]
@kamilmysliwiec, I'm looking into this and trying to fix my bug, but I am a bit confused. Even if I do `nest start --debug`, options still looks like this: ```[
  { name: 'config', value: undefined },
  { name: 'webpack', value: undefined },
  { name: 'debug', value: undefined },
  { name: 'watch', value: false },
  { name: 'watchAssets', value: false },
  { name: 'path', value: undefined },
  { name: 'webpackPath', value: undefined },
  { name: 'exec', value: undefined },
  { name: 'preserveWatchOutput', value: false }
]```

How does it work?
